README.md

```diff
@@ -72,6 +72,8 @@
 
 - **[Creating a data model]**
 - **[Process data in model]**
+- **[Add validations to model]**
+- **[Add Rspec tests to model]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![Screenshot from 2023-07-13 19-02-01](https://github.com/Atifzada/blog_app/assets/83042742/b7c0f522-6950-4541-92bd-674107d1536c)
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

- [ ] To make your README looks more professional and descriptive, kindly ensure that you use the appropriate link to your `Live Demo` section as the current doesn't open the deployed application. As your app is not deployed yet, I suggest to completely remove the `Live Demo` section from now, you can add it later with the working link once deployed. 
You should ensure that you remove other empty section(s) such as the `Deployment`,... to improve your documentation.

Reply:
Thank you for the suggestions surely I will fix it